x-pack/platform/plugins/shared/streams_app/public/components/data_management/condition_editor/index.tsx

```diff
@@ -252,7 +252,7 @@ function FilterForm(props: {
         onChange={handleOperatorChange}
         disabled={disabled}
       />
-      {typeof value === 'string' ? (
+      {typeof value === 'string' || typeof value === 'number' ? (
```

Comment:
@flash1293 I might be missing something here
Why are we not showing conditions with `typeof value === 'number'`?
I added this one here because I ran into this case while testing manually

Reply:
I think @Kerry350 changed that in https://github.com/elastic/kibana/pull/230064/files#diff-e26bd006a23ec0f7200a2122bbd67620711a574586123faea7fd1e40c3fb4815R233

I'm not sure how consequential it is, because you when using the UI it will always be a string eventually. But I'll let @Kerry350 answer

Reply:
Thanks for your input @Kerry350. I just reverted this change in [7d40255](https://github.com/elastic/kibana/pull/235381/commits/7d40255b88b4cd9cc8bedb5c3547d0dc9cf71c5b) and handled the `boolean` case as well.